packages/vite/src/node/ssr/runtime/__tests__/server-runtime.spec.ts

```diff
@@ -141,63 +140,61 @@ describe('vite-runtime initialization', async () => {
     `)
     // subsequent imports of the same external package should not throw if imports are correct
     await expect(
-      runtime.executeUrl('/fixtures/cjs-external-existing.js'),
+      runner.import('/fixtures/cjs-external-existing.js'),
     ).resolves.toMatchObject({
       result: 'world',
     })
   })
 
-  it('importing external esm library checks exports', async ({ runtime }) => {
+  it('importing external esm library checks exports', async ({ runner }) => {
     await expect(() =>
-      runtime.executeUrl('/fixtures/esm-external-non-existing.js'),
+      runner.import('/fixtures/esm-external-non-existing.js'),
     ).rejects.toThrowErrorMatchingInlineSnapshot(
       `[SyntaxError: [vite] The requested module '@vitejs/esm-external' does not provide an export named 'nonExisting']`,
     )
     // subsequent imports of the same external package should not throw if imports are correct
     await expect(
-      runtime.executeUrl('/fixtures/esm-external-existing.js'),
+      runner.import('/fixtures/esm-external-existing.js'),
     ).resolves.toMatchObject({
       result: 'world',
     })
   })
 
-  it("dynamic import doesn't produce duplicates", async ({ runtime }) => {
-    const mod = await runtime.executeUrl('/fixtures/dynamic-import.js')
+  it("dynamic import doesn't produce duplicates", async ({ runner }) => {
+    const mod = await runner.import('/fixtures/dynamic-import.js')
     const modules = await mod.initialize()
     // toBe checks that objects are actually the same, not just structually
     // using toEqual here would be a mistake because it chesk the structural difference
-    expect(modules.static).toBe(modules.dynamicProcessed)
-    expect(modules.static).toBe(modules.dynamicRelative)
-    expect(modules.static).toBe(modules.dynamicAbsolute)
-    expect(modules.static).toBe(modules.dynamicAbsoluteExtension)
+    expect(modules.static === modules.dynamicRelative).toBe(true)
+    expect(modules.static === modules.dynamicRelative).toBe(true)
+    expect(modules.static === modules.dynamicAbsolute).toBe(true)
+    expect(modules.static === modules.dynamicAbsoluteExtension).toBe(true)
+    expect(modules.static === modules.dynamicAbsoluteFull).toBe(true)
```

Comment:
Why was this changed to `===`?

Reply:
We throw an error in the `import.meta.env` getter because it should be statically replaced, and Vitest cannot build a diff because of that

Reply:
Updated it to use a `simple` module instead that doesn't export `import.meta`